config/forms/2022_2023.json

```diff
@@ -8460,7 +8443,8 @@
                       }
                     ]
                   },
-                  "header": "Are you sure this is correct?",
+                  "header": "This rent is higher than expected for this property type, in this area. Check:",
+                  "hint_text": "<ul><li>the decimal point is not missing (£X.XX)</li><li>the frequency is correct, for example weekly, monthly</li><li>the rent type is correct, for example affordable or social rent</li></ul><p>Once checked, confirm below.</p>",
```

Comment:
as this is identical to the string above, I would ideally like this to be in the yml file. For design reasons it should be the hint text though, so as I understand it it will just have to be here as hint text can only be a plain string at this stage.